src/pages/InquiriesPage/index.tsx

```diff
@@ -6,29 +6,34 @@ import TableCell from '@mui/material/TableCell';
 import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
-import { useEffect, useState } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { getInquiryData } from '~/apis/inquiry';
 import { PaginationBox } from '~/components/PaginationBox';
 import { INQUIRY_CATEGORIES } from '~/constants/INQUIRY_CATEGORIES';
 import { clickableStyle } from '~/styles/common';
 import { InquiryData, InquiryParams } from '~/types/api.types';
 import { InquiryCategories } from '~/types/inquiry.types';
+import { InquiriesDialog } from './InquiriesDialog';
 
 export const InquiriesPage: React.FC = () => {
   const [inquiryList, setInquiryList] = useState<InquiryData>();
+  const [isOpenDialog, setIsOpenDialog] = useState<boolean>(false);
+  const [selectedId, setSelectedId] = useState<number | null>(null);
```

Comment:
selectedId가 있다면 isOpenDiaglog가 true가 되는 것 같아서 하나만 있어도 될 것 같다는 생각이 듭니다.

Reply:
넵 id가 있는 경우 모달이 열리긴 하는데 그래도 명확하게 각자의 역할에 맞는 값을 주고 싶었습니다

Reply:
분리되지 않고 따라가는 값이라면 아래처럼 상태가 아닌 값으로 관리하면 상태를 줄일 수 있을 것 같아요. 최근에 봤던 리액트 문서에서 상태는 최소한으로 갖는 것을 권장한다고 봤어요. [State 구조 선택하기 - State의 모순 피하기](https://ko.react.dev/learn/choosing-the-state-structure#avoid-contradictions-in-state)

```suggestion
  const [selectedId, setSelectedId] = useState<number | null>(null);
  const isOpenDialog = !!selectedId;
```

Reply:
수정완 [c317bf0](https://github.com/jazz-meet/jazz-meet-admin/pull/15/commits/c317bf0e60c179ac9409e04a158495fcda2091da)